docs/source/guide/export.md

```diff
@@ -38,10 +38,24 @@ Use the following steps to export data and annotations from the Label Studio UI.
 
 If the export times out, see how to [export snapshots using the SDK](https://labelstud.io/sdk/project.html#label_studio_sdk.project.Project.export_snapshot_create) or [API](#Export-snapshots-using-the-API).
 
+### Export all tasks including tasks without annotations
+
+Label Studio opensource exports tasks with annotations only by default. If you need to easily export all tasks including tasks without annotations, you can call this [Easy Export API](https://api.labelstud.io/#operation/api_projects_export_read) with query param `download_all_tasks=true`, e.g.:
```

Comment:
Label Studio open source exports tasks with annotations only by default. If you want to easily export all tasks including tasks without annotations, you can call  the [Easy Export API](https://api.labelstud.io/#operation/api_projects_export_read) with query param `download_all_tasks=true`. For example: